### PR TITLE
Update main.js

### PR DESCRIPTION
**Pull request by M-Eldin**
Update 1

- cleaned up code 
- added "Life Bar" and Life counter
- added loop for multiple playability
- added final death message

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -3,27 +3,60 @@ var findVerbs = require("./verbs.js");
 var findAdjective = require("./adjectives.js");
 var findNoun = require("./nouns.js");
 
-//defines chance of survival
-let Random = () => {
-  return Math.floor(Math.random() * 2);
-};
+//Life counter
+let Life = ["♥", "♥", "♥"];
```
Comment (RussellRansom) [inline main.js:7]: Little concerned about the encoding of these characters. Are these default UTF-8 chars?

Comment (RussellRansom) [reply main.js:7]: Typically we don't name instance variables with capital letters, that is reserved for classes/objects

File: main.js [hunk 2]
```diff
@@ -3,27 +3,60 @@ var findVerbs = require("./verbs.js");
...
 
-let Odds = () => {
+  //Assign randomness
+  Random = () => {
+    return Math.floor(Math.random() * 2);
+  };
+  //Assign chance of survival based on randomness
   if (Random() === 0) {
-    return "alive";
+    var Odds = "alive";
```
Comment (RussellRansom) [inline main.js:19]: var starts with a capital letter. Also, could use a better naming scheme. Odds doesn't represent the odds of being alive or dead, it represents that actual state of alive or dead. Something like "lifeStatus" would be more informative to a reader

File: main.js [hunk 3]
```diff
@@ -3,27 +3,60 @@ var findVerbs = require("./verbs.js");
...
+      " " +
+      findNoun() +
+      " and decide to " +
+      findVerbs() +
+      " it, you are " +
+      Odds +
+      " after the encounter"
+  );
+
+  //display outcome of each loop
```
Comment (RussellRansom) [inline main.js:36]: In ReRoll and Loss, you're checking the value of Odds multiple times. Is there a way to only do one check for "alive" or "dead" and perform all the needed operations in one swoop?

File: main.js [hunk 4]
```diff
@@ -3,27 +3,60 @@ var findVerbs = require("./verbs.js");
...
+    if (Odds === "alive") {
+      console.log("You live another day!");
+    } else {
+      console.log("You lose a life");
+    }
+  };
+  ReRoll();
+
+  //display final loop
+  Loss = () => {
```
Comment (RussellRansom) [inline main.js:47]: I like the idea of pulling some of these functions out of the body of Odds to make the logic simpler, as you did with the Random function. What if you pulled Loss out of here and defined it in the larger scope as a function. You could pass the current Life total into the function and return the new life total.

File: main.js [hunk 5]
```diff
@@ -3,27 +3,60 @@ var findVerbs = require("./verbs.js");
 var findAdjective = require("./adjectives.js");
```
Comment (RussellRansom) [inline main.js:3]: Where are these imports? Am I just not seeing them in the repo or did you not commit them?

**Review by RussellRansom — CHANGES_REQUESTED** [review]
A few changes to be made - an awesome start though. Looks really good

File: main.js (hunk 5 above)
Comment (M-Eldin) [reply main.js:3]: the cloned main branch contains the required .js imports

File: main.js (hunk 1 above)
Comment (M-Eldin) [reply main.js:7]: I know I know I sacrificed for a stylistic choice :( it looks good and the hearts show up green

Comment (M-Eldin) [reply main.js:7]: Capture

File: main.js (hunk 2 above)
Comment (M-Eldin) [reply main.js:19]: great suggestion. Changed

File: main.js (hunk 3 above)
Comment (M-Eldin) [reply main.js:36]: awesome. Done and done, simplified the code much more

**Pull request merged**